packages/nitro-server/test/route-rules-compiler.test.ts

```
@@ -0,0 +1,85 @@
+
+  it('matches the rules nitro would match under a base URL', () => {
+    const nitro = matchWithNitro(rules, '/base')
+    const nuxt = matchWithNuxt(rules, '/base')
+    for (const path of paths) {
+      expect.soft(nuxt('/base' + path), path).toEqual(nitro('/base' + path))
+    }
+  })
+
+  it('compiles a lone catch-all to its rules', () => {
+    const router = createRouteRulesRouter([{ route: '/**', method: '', data: { ssr: false } }], '')
+    const match = eval(router.compileToString({ matchAll: true })) as (method: string, path: string) => { data: unknown }[]
```

Biome (2.5.8)

[error] eval() exposes to security risks and performance issues.

(lint/security/noGlobalEval)